Snip/Players/mpsyt.cs

```diff
@@ -65,37 +61,39 @@ public override void Unload()
 
         public override void ChangeToNextTrack()
         {
```

Comment:
All of the keys listed here can be added to the Snip.cs file under ToggleHotkeys() and KeyboardHook_KeyPressed() methods.